### DISCUSSION TITLE
Disable vibrations with Sculk Sensor

### DISCUSSION THREAD

**Discussion by NoltoxGit** [PurpurMC/Purpur#989] [__Dismissed__] [answered]
Hello !

I have an idea of additions that would allow to deactivate something that is not yet used for the 1.18 versions which is the vibrations with the Sculk Sensor blocks.

In my case where I make a creative server with plots, I have players (despite the restrictions) who continue to make the server lag by putting a lot on a single plot. 
I don't think I'm the only one who still doesn't find this block useful in this version except for making the server crash/massive lag.

I'm open to any answers,
Ty, 
~ Noltox ~

**Comment by encode42** [comment, marked as answer]
This is definitely the job for a plugin, and there's many out there that already have the ability to do this. If you've already tried using one of these plugins, it's most likely misconfigued.
